pkg/util/conditionStatus.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import (
+	"github.com/replicatedhq/embedded-cluster-kinds/apis/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CheckConditionStatus(inStat v1beta1.InstallationStatus, conditionName string) metav1.ConditionStatus {
```

Comment:
i added a k8sutil package. wanna move this there or move my stuff here?

Reply:
I can move it there